docs/docs/introduction.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
```

Comment:
Suggestions:
- add a link to README.md from here, something like "For full inspiration and Vision"
- in future, we can add Roadmap
- would be great to link to the Core Team and All Contributors section in the README as well